sdk/mobile-center-distribute/src/main/java/com/microsoft/azure/mobile/distribute/ReleaseDetails.java

```diff
@@ -85,14 +92,15 @@ static ReleaseDetails parse(String json) throws JSONException {
         if (scheme == null || !scheme.startsWith("http")) {
             throw new JSONException("Invalid download_url scheme.");
         }
+        releaseDetails.mandatoryUpdate = object.getBoolean(MANDATORY_UPDATE);
         releaseDetails.releaseHash = object.getJSONArray(PACKAGE_HASHES).getString(0);
         return releaseDetails;
     }
 
     /**
      * Get the id value.
      *
-     * @return the id value.
+     * @return the id value
```

Comment:
Like everywhere else. We should fix it everywhere else instead but didn't have time for that.